cylc/flow/network/schema.py

```diff
@@ -2180,20 +2180,24 @@ class Arguments(TaskMutation.Arguments, FlowMutationArguments):
 class Trigger(Mutation, TaskMutation):
     class Meta:
         description = sstrip('''
-            Manually trigger tasks.
+            Manually trigger tasks, even in a paused workflow.
 
-            Warning: waiting tasks that are queue-limited will be queued if
-            triggered, to submit as normal when released by the queue; queued
-            tasks will submit immediately if triggered, even if that violates
-            the queue limit (so you may need to trigger a queue-limited task
-            twice to get it to submit immediately).
+            Triggering an unqueued task queues it, to run when queue-released.
+            Triggering a queued task runs it now regardless of queue limiting.
+
+            The "on resume" option waits for a paused workflow to be resumed.
```

Comment:
What exactly don't you like about the original? 

Well, "a task that is not yet queued" is technically not "queue-limited".  We, or at least I, have always taken "queue-limited" to mean what happens when the queue's active task limit has been reached, i.e. when a task is ready to run but the queue is holding it back. 

I was trying to avoid making that distinction, as something potentially requiring user thought, by simply saying that triggering a task queues it, to run when released by the queue - which is immediately if queue-limiting is not in effect, or later if it is in effect.

Reply:
How about this:
```suggestion
            Triggering a task that is not yet queued, queues it.
               
            Triggering a queued task runs it immediately.
                        
            Queues release tasks to run when the count of active tasks
            drops below the queue limit. So, depending on activity, you
            may need to trigger a task twice to make it run immediately. 
```

Reply:
Ok, small amendment to your suggestion:
```suggestion
            Triggering a task that is not yet queued will queue it.
               
            Triggering a queued task runs it immediately.
                        
            Queues release tasks to run when their active task count
            drops below the queue limit. So, depending on the task count, you
            may need to trigger a task twice to make it run immediately. 
```